### PR TITLE
New fields for stations API 

### PR DESCRIPTION
**Pull request by reger**

### CODE REVIEW

File: source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb [hunk 1]
```diff
@@ -16,7 +16,7 @@
       </td>
       <td class="doc-parameter-description">
         <p>The type of alternative fuel the station provides. Fuel types are given as code values as described below:</p>
-        <table border="0" cellpadding="0" cellspacing="0" class="doc-parameter-options">
+        <table border="0" cellpadding="0" cellspacing="0" class="table table-bordered">
```
Comment (GUI) [inline source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb]: Thanks for finding these table style inconsistencies. If we're making these consistent, though, I think we should also make the other sub-tables up top in the requests area consistent too: https://developer.nrel.gov/docs/transportation/alt-fuel-stations-v1/all/#request-parameters https://github.com/NREL/developer.nrel.gov/blob/new-fields/source/docs/transportation/alt-fuel-stations-v1/_request_params_common2.html.md.erb#L17

File: source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb [hunk 2]
```diff
@@ -803,6 +1220,263 @@
...
+          <thead>
+            <tr>
+              <th scope="col">Value</th>
+              <th scope="col">Description</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <th scope="row">AIRPORT</th>
+              <td>Airport</td>
```
[the remark this answers is not in the document]
Comment (reger) [reply source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb]: good idea, let's add a story in privotal for this, if there isn't one already

File: source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb [hunk 3]
```diff
@@ -112,84 +112,167 @@ The individual fuel station records are returned in no specific order. Each stat
...
-      "updated_at": "2013-02-14T18:43:05Z"
+      "lpg_nozzle_types": null,
+      "ng_fill_type_code": null,
+      "ng_psi": null,
+      "ng_vehicle_class": null,
+      "access_days_time_fr": null,
+      "intersection_directions_fr": null,
+      "bd_blends_fr": null,
+      "groups_with_access_code_fr": "Public",
+      "ev_pricing_fr": null
```
[the remark this answers is not in the document]
Comment (reger) [reply source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb]: Done. For now I just copied the english field documentation, except for `groups_with_access_code_fr`. I put the French versions of the values we output there.

File: source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb [hunk 4]
```diff
@@ -112,84 +112,167 @@ The individual fuel station records are returned in no specific order. Each stat
...
       "state": "CA",
-      "ng_fill_type_code": null,
-      "ng_psi": null,
-      "ng_vehicle_class": null,
-      "e85_blender_pump": false,
-      "ev_level1_evse_num": null,
-      "ev_level2_evse_num": null,
+      "street_address": "1015 L St",
+      "zip": "95814",
+      "country": "US",
```
[the remark this answers is not in the document]
Comment (reger) [reply source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb]: Done.

File: source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb [hunk 5]
```diff
@@ -112,84 +112,167 @@ The individual fuel station records are returned in no specific order. Each stat
...
+    }
+  },
   "fuel_stations": [
     {
-      "access_days_time": "24 hours daily; also accept Propel Clean Drive fleet card",
-      "bd_blends": null,
-      "cards_accepted": "A D M V FuelMan Voyager Wright_Exp",
-      "city": "Fremont",
-      "date_last_confirmed": "2013-01-31",
+      "access_code": "public",
```
[the remark this answers is not in the document]
Comment (reger) [reply source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb]: Done.

File: source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb [hunk 6]
```diff
@@ -112,84 +112,167 @@ The individual fuel station records are returned in no specific order. Each stat
...
+      "access_days_time": "Garage business hours; pay lot",
+      "access_detail_code": null,
```
[the remark this answers is not in the document]
Comment (reger) [reply source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb]: Done.

File: source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb [hunk 7]
```diff
@@ -556,6 +622,241 @@
...
+      <th class="doc-parameter-name" scope="row">cng_dispenser_num</th>
+      <td class="doc-parameter-value">
+        <div class="doc-parameter-value-field">
+          <strong>Type:</strong> integer
+        </div>
+      </td>
+      <td class="doc-parameter-description">For CNG stations, the number of CNG dispensers installed.</td>
+    </tr>
+    <tr>
+      <th class="doc-parameter-name" scope="row">cng_fill_type_code</th>
```
[the remark this answers is not in the document]
Comment (reger) [reply source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb]: Done.

File: source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb [hunk 8]
```diff
@@ -112,84 +112,167 @@ The individual fuel station records are returned in no specific order. Each stat
 
 ```json
 {
+  "station_locator_url": "http://www.afdc.energy.gov/afdc/locator/stations/",
+  "total_results": 5980,
+  "station_counts": {
```
[the remark this answers is not in the document]
Comment (reger) [reply source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb]: @GUI Can you tackle this? Not sure how you'd want to explain it.

File: source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb [hunk 9]
```diff
@@ -112,84 +112,167 @@ The individual fuel station records are returned in no specific order. Each stat
...
+      ],
       "ev_dc_fast_num": null,
-      "ev_other_evse": null,
+      "ev_level1_evse_num": 2,
+      "ev_level2_evse_num": null,
       "ev_network": null,
       "ev_network_web": null,
+      "ev_other_evse": "3 SP Inductive",
+      "ev_pricing": null,
+      "hy_is_retail": null,
```
[the remark this answers is not in the document]
Comment (reger) [reply source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb]: Done.

File: source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb (hunk 5 above)
[the remark this answers is not in the document]
Comment (reger) [reply source/docs/transportation/alt-fuel-stations-v1/all.html.md.erb]: Updated example outputs. Added `ev_network_ids` description.

File: source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb (hunk 1 above)
Comment (reger) [reply source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb]: Updated table styles in request params

File: source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb [hunk 10]
```diff
@@ -556,6 +622,241 @@
...
+      <th class="doc-parameter-name" scope="row">cng_total_storage</th>
+      <td class="doc-parameter-value">
+        <div class="doc-parameter-value-field">
+          <strong>Type:</strong> integer
+        </div>
+      </td>
+      <td class="doc-parameter-description">For CNG stations, the total storage capacity, measured in standard cubic feet (scf).</td>
+    </tr>
+    <tr>
+      <th class="doc-parameter-name" scope="row">cng_vehicle_class</th>
```
[the remark this answers is not in the document]
Comment (reger) [reply source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb]: Done.

File: source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb [hunk 11]
```diff
@@ -556,6 +622,241 @@
...
+            <tr>
+              <th scope="row">B</th>
+              <td>Both: quick fill and timed fill</td>
+            </tr>
+          </tbody>
+        </table>
+      </td>
+    </tr>
+    <tr>
+      <th class="doc-parameter-name" scope="row">cng_psi</th>
```
[the remark this answers is not in the document]
Comment (reger) [reply source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb]: Done.

**Comment by reger** [comment]
@GUI found a few typos and fixed, I also appended the country to the end of the federal agency name in the lists since we have both US and CA agencies

**Pull request merged**